st2common/bin/st2-run-pack-tests

```diff
@@ -104,6 +107,14 @@ while getopts ":p:f:xjvct" o; do
         t)
             ENABLE_TIMING=true
             ;;
+        3)
+            PYTHON_3=true
+            if [ -z "$ST2_REPO_PATH" ]; then
+                echo "must set ST2_REPO_PATH variable"
```

Comment:
A clearer error message explaining what the `ST2_REPO_PATH` variable is used for will help people set the correct value. e.g. `Environment variable ST2_REPO_PATH must be set to your st2 installation or st2 git repository for pack tests to function correctly.`